pages/common/clear.md

```diff
@@ -5,3 +5,15 @@
 - Clear the screen (equivalent to typing Control-L when using the bash shell):
 
 `clear`
+
+- Clear the screen but keep the terminal's scrollback buffer:
+
+`clear -x`
+
+- Indicate the type of terminal to clean (normally this option is unnecessary because it is taken from the environment variable `TERM`):
+
+`clear -T {type_of_terminal}`
+
+- Show the version of `ncurses` used by `clear`.
```

Comment:
Is this really necessary? Seems unneeded to absolute majority of the users.

Reply:
Yes, that's true. But it is a command with just a few parameters. Do you think it would be necessary to remove it?

Reply:
You can wait for the opinions of others, but I'd keep only the useful examples. :wink:

Reply:
As we are waiting for more opinions in the above issue, we can wait.

Reply:
There's no harm in including it, I guess - since there isn't anything else we can include on the  page.

Reply:
So could it being marked as resolved? Or should we wait for more opinions about this?

Reply:
I think we could maintain the example and if we have problems exceeding the examples limit, then this example would be the first one to be removed.